packages/chain-helpers/src/blockchain/Blockchain.ts

```diff
@@ -139,11 +139,9 @@ export async function submitSignedTx(
 
   function handleDisconnect(): void {
     const result = new SubmittableResult({
-      events: latestResult.events || [],
+      events: latestResult.events,
```

Comment:
That change looks not correct to me. `latestResult` can be declared, but not instantiated, before reaching `handleDisconnect`. Same with the status line below.